plugin/compile-scss.js

```diff
@@ -4,38 +4,16 @@ const path = Plugin.path;
 const fs = Plugin.fs;
 
 const compileSass = promisify(sass.render);
-let _includePaths = [];
 const rootDir = (process.env.PWD || process.cwd()) + "/";
 
+const config = _getConfig('scss-config.json');
+const _includePaths = Array.isArray(config['includePaths']) ? config['includePaths'] : [];
```

Comment:
This is now just loading the config one time and setting `_includePaths` right away. Should fix all problems with it.